app/controllers/settings/password.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  actions: {
+    changePassword(passwordData) {
+      this.set('isLoading', true);
+      let payload = {
+        'data': {
+          'old-password' : passwordData.passwordCurrent,
+          'new-password' : passwordData.passwordNew
+        }
+      };
+      this.get('loader')
+        .post('/auth/change-password', payload)
+        .then(data => {
+          if (data.error) {
```

Comment:
@niranjan94 There is a strange thing happening here. When I make request manually throught postman, I get the response 400 which ideally should throw an error and should be handled in catch part.
![change-password](https://user-images.githubusercontent.com/17925669/29569896-82a8fc72-8772-11e7-92e9-4961ea6df391.png)
When I try to log the error:
```
  .catch((error) => {
          console.log('Error :', error);
          this.set('isLoading', false);
          this.setProperties({ 'passwordCurrent': '', 'passwordNew': '', 'passwordRepeat': '' });
          this.get('notify').error(this.l10n.t('An unexpected error occured. Password did not change.'));
        });
```
I get:
![change-password2](https://user-images.githubusercontent.com/17925669/29569984-c78a95b2-8772-11e7-866a-135f76776422.png)

However in my networks tab I get the response returned by server:
![change-pass](https://user-images.githubusercontent.com/17925669/29570033-e96be97e-8772-11e7-9f26-8f134f6bc0dc.png)

Not getting why the server 400 response is not being handled in catch part.